design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
 
-|      | Given                                                                                                                | When                                                                | Then                                                                                     | Manual / UT / Integration | Status  |
-|------|----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|------------------------------------------------------------------------------------------|---------------------------|---------|
-| 13-1 | A host with no Azure-specific identification in its annotations or they are empty                                    | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `RoleMissingAnnotations` message appears in log           | Integration               | [ ]     |
-| 13-2 | A host with Subscription ID + Resource Group + User Assigned Identity in its annotations                             | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | [ ]     |
-| 13-3 | A host with Subscription ID + Resource Group + System Assigned Identity in its annotations                           | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | [ ]     |
-| 13-4 | A host with Subscription ID + Resource Group in its annotations                                                      | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | [ ]     |
-| 13-5 | A host with Subscription ID OR Resource Group in its annotations                                                     | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `RoleMissingAnnotations` message appears in log           | Integration               | [ ]     |
-| 13-6 | A host with Subscription ID + Resource Group + User Assigned Identity + System Assigned Identity in its annotations  | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `IdentityAssignedInvalid` message appears in log          | Integration               | [ ]     |
+|      | Given                                                                                                                | When                                                                | Then                                                                                     | Manual / UT / Integration | Status                            |Core Test|
```

Comment:
I think that all this table should be a core test as it's part of the main flow of authn-azure

Reply:
I think that one scenario of success and failure is enough , we have UT on those tests as well.

Reply:
ok, i'm with you.